users.js

```diff
@@ -120,29 +137,35 @@ module.exports = function (options) {
 
       args.mailingList = (args.mailingList) ? 1 : 0;
 
-      seneca.act({role: 'user', cmd: 'register'}, args, function (err, registerResponse) {
+      checkPassword(args, function (err, args) {
         if (err) return done(err);
-        if (!registerResponse.ok) {
-          return done(null, registerResponse);
+        if (typeof args.ok !== 'undefined' && !args.ok) {
```

Comment:
Makes sense. Basically make sure it's always true or false and never anything else.